requirements.txt

```diff
@@ -1,7 +1,7 @@
 docutils == 0.17
-sphinx == 4.3.2
-sphinx-copybutton == 0.4.0
-sphinx-design == 0.0.13
+sphinx == 5.1.1
```

Comment:
Might drop this back.

Sphinx 5.1 wants DocUtils 0.20.0, but MyST wants 0.19.0

I'm not seeing anything we can use in newer versions of sphinx, it mostly seems autodoc related. Might get us up to 4.4.0 and DocUtils 0.19.0 and freeze it there.

Also need to potentially fork DocUtils 0.19.0 and patch the code for the Include directive to return better errors around missing keys.